locales/pt.json

```diff
@@ -1,215 +1,215 @@
- 	"Starting Blockchain node in another process": "Starting Blockchain node in another process",
- 	"Blockchain node is ready": "Blockchain node is ready",
- 	"Storage process for {{storageName}} ended before the end of this process. Code: {{code}}": "Storage process for {{storageName}} ended before the end of this process. Code: {{code}}",
- 	"ipfs process started": "ipfs process started",
- 	"Compiling...": "Compiling...",
- 	"Building...": "Building...",
- 	"gas at the price of": "gas at the price of",
- 	"Wei, estimated cost:": "Wei, estimated cost:",
- 	"using": "using",
- 	"Embark is building, please wait...": "Embark is building, please wait...",
- 	"instantiated js-ipfs object configured to the current environment (available if ipfs is enabled)": "instantiated js-ipfs object configured to the current environment (available if ipfs is enabled)",
- 	"Initializing Embark Template...": "Initializing Embark Template...",
- 	"Name your app (default is %s):": "Name your app (default is %s):",
- 	"Unrecognized balance string \"%s\"": "Unrecognized balance string \"%s\"",
- 	"{{className}} has a dependency on {{dependencyName}}": "{{className}} has a dependency on {{dependencyName}}",
- 	", but it is not set to deploy. It could be an interface.": ", but it is not set to deploy. It could be an interface.",
- 	"Done. %s generated": "Done. %s generated"
-}
+	"New Application": "Nova Aplicacao",
+	"Contract Name": "Contracto",
+	"Address": "Endereço",
+	"Status": "Estado",
+	"Embark Blockchain Using: %s": "Embark Blockchain esta usando o commando: %s",
+	"running: %s": "executando: %s",
+	"already initialized": "já está inicializado",
+	"create a barebones project meant only for contract development": "criar um projeto vazio destinado apenas ao desenvolvimento de contratos",
+	"loading solc compiler": "carregando o compilador solc",
+	"Welcome to Embark": "Bem-vindo ao Embark",
+	"possible commands are:": "comandos possíveis são:",
+	"display versions in use for libraries and tools like web3 and solc": "lista versões em uso para bibliotecas e ferramentas como web3 e solc",
+	"instantiated web3.js object configured to the current environment": "objeto web3.js instanciado configurado para o ambiente atual",
+	"to immediatly exit (alias: exit)": "para sair imediatamente (atalho: exit)",
+	"The web3 object and the interfaces for the deployed contracts and their methods are also available": "O objeto web3 e as interfaces para os contratos implantados e seus métodos também estão disponíveis",
+	"create a working dapp with a SimpleStorage contract": "Cria uma dapp funcional com o contrato SimpleStorage",
+	"filename to output logs (default: none)": "ficheiro/arquivo para saída dos logs (predefinido: none)",
+	"level of logging to display": "nivel do log",
+	"deploy and build dapp at ": "Publica os contractos e constroi a applicacao em ",
+	"port to run the dev webserver (default: %s)": "porta para correr o servidor wen para desenvolvimento (default: %s)",
```

Comment:
`wen` -> `web`

Reply:
Woops, `Find and Replace` failed :disappointed: